### ISSUE TITLE
How to tell a transaction is USDt transaction?

### ISSUE THREAD

**Issue by chaoyebugao** [OmniLayer/omnicore#1063]
I am doing the deposit business. And I am have same addresses in BTC and USDt. When I receive a transaction from MainChain. I have to tell is BTC or USDt transaction. But how?

**Comment by pundoo** [comment]
Yes. It's the `OP_RETURN` that defines omni layer (USDT) transaction.
